### PR TITLE
Updates to the documentation Contribution section

### PR DESCRIPTION
**Pull request by alwasega**

### CODE REVIEW

**Comment by alwasega** [comment]
@minrk, please check this draft PR in which I have made slight changes to the `Contributing` section.

File: docs/source/contributing/index.md [hunk 1]
```diff
@@ -1,20 +1,80 @@
...
+
+Here are the people who keep JupyterHub running and updated.
+
+```{toctree}
+:maxdepth: 1
+
+contributor-list
+```
+
+## Communication channels
```
Comment (minrk) [inline docs/source/contributing/index.md:22]: I don't think we should have new headings and separate toctrees for each single document. I think a single toctree is best, and we should make sure to get this information from the headings we get from those documents.

Comment (alwasega) [reply docs/source/contributing/index.md:22]: Okay. Let me revert to the previous headings and toctree.

**Comment by alwasega** [comment]
Thank you, @sgibson91.

**Pull request merged**